api/api.yml

```diff
@@ -0,0 +1,113 @@
+openapi: 3.0.0
```

Comment:
Eventually, we'll ref YML from our common API repo instead of maintaining a separate copy here. The copy is useful in the short-term to decouple the two projects.